File: legacy-runtime/src/main/java/com/xiaomi/clientreport/data/EventClientReport.java
package com.xiaomi.clientreport.data;

import com.xiaomi.channel.commonutils.logger.MyLog;
import org.json.JSONException;
import org.json.JSONObject;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/clientreport/data/EventClientReport.class */
public class EventClientReport extends BaseClientReport {
    public String eventContent;
    public String eventId;
    public long eventTime;
    public int eventType;

    public static EventClientReport getBlankInstance() {
        return new EventClientReport();
    }

    @Override // com.xiaomi.clientreport.data.BaseClientReport
    public JSONObject toJson() {
        try {
            JSONObject json = super.toJson();
            if (json == null) {
                return null;
            }
            json.put("eventId", this.eventId);
            json.put("eventType", this.eventType);
            json.put("eventTime", this.eventTime);
            String str = this.eventContent;
            String str2 = str;
            if (str == null) {
                str2 = "";
            }
            json.put("eventContent", str2);
            return json;
        } catch (JSONException e) {
            MyLog.e(e);
            return null;
        }
    }

    @Override // com.xiaomi.clientreport.data.BaseClientReport
    public String toJsonString() {
        return super.toJsonString();
    }
}

File: legacy/src/main/java/com/xiaomi/clientreport/data/EventClientReport.kt
package com.xiaomi.clientreport.data
import io.github.magisk317.mipush.protocol.model.*

import com.xiaomi.channel.commonutils.logger.MyLog
import org.json.JSONException
import org.json.JSONObject

class EventClientReport : BaseClientReport() {
    @JvmField var eventContent: String? = null
    @JvmField var eventId: String? = null
    @JvmField var eventTime: Long = 0
    @JvmField var eventType: Int = 0

    companion object {
        @JvmStatic
        fun getBlankInstance(): EventClientReport = EventClientReport()
    }

    override fun toJson(): JSONObject? {
        return try {
            val json = super.toJson() ?: return null
            json.put("eventId", eventId)
            json.put("eventType", eventType)
            json.put("eventTime", eventTime)
            json.put("eventContent", eventContent ?: "")
            json
        } catch (e: JSONException) {
            MyLog.e(e)
            null
        }
    }

    override fun toJsonString(): String = super.toJsonString()
}
